crates/storage/src/key.rs

```diff
@@ -324,6 +324,89 @@ where
     }
 }
 
+impl<T> PrimaryKey for Signed<T>
+where
+    T: PrimaryKey<Output = T> + Sub<Output = T> + Copy,
+    T: NumberConst,
+{
+    type Output = Self;
+    type Prefix = ();
+    type Suffix = ();
+
+    const KEY_ELEMS: u8 = 1;
+
+    fn raw_keys(&self) -> Vec<Cow<[u8]>> {
+        let (mut keys, sign) = match self.is_negative() {
+            true => {
+                let dif = T::MAX - *self.inner();
+
+                let keys = dif
+                    .raw_keys()
+                    .into_iter()
+                    .map(|val: Cow<[u8]>| Cow::Owned(val.into_owned()))
```

Comment:
This type notation can be removed, since the compiler can infer it. 

```suggestion
                    .map(|val| Cow::Owned(val.into_owned()))
```